src/redux/actions/ResetPasswordAction.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import * as ActionTypes from "../action_types";
+import API_BASE_URL from "./index";
+
+/**
+ * error from forgot password request
+ *
+ * @param {string} error
+ * @return {object} type and error
+ */
+
```

Comment:
You forgot to delete the empty lines on this page

Reply:
I have deleted it.